changelog/1316.bugfix.rst

```diff
@@ -0,0 +1 @@
+TestReport.longrepr is always instance of ReprExceptionInfo. Previously it was string when test failed with pytrace=False.
```

Comment:
```suggestion
``TestReport.longrepr`` is now always an instance of ``ReprExceptionInfo``. Previously it was a ``str`` when a test failed ``pytest.fail(..., pytrace=False)``.
```

This is also a breaking change, so please rename the file to `1316.breaking.rst`. 👍